### PR TITLE
MSS-1266: Make webTitle the body of a liveblog notification and include a series tag in the title

### PR DESCRIPTION
**Pull request by frankie297**

### CODE REVIEW

File: src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala [hunk 1]
```diff
@@ -53,8 +53,8 @@ trait ContentAlertPayloadBuilder extends Logging {
 
   def buildPayLoad(content: Content, keyEvent: KeyEvent): ContentAlertPayload = {
     ContentAlertPayload(
-      title = Some(s"Liveblog update: ${keyEvent.title.getOrElse(content.webTitle)}"),
-      message = if (keyEvent.title.isDefined) Some(content.webTitle) else None,
+      title = Some(s"Liveblog update"),
+      message = if (keyEvent.title.isDefined) Some(s"${keyEvent.title.getOrElse(content.webTitle)}") else None,
```
Comment (alexduf) [inline src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala]: I think the if condition can be removed and always be `Some(s"${keyEvent.title.getOrElse(content.webTitle)}")`

Comment (frankie297) [reply src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala]: I tried this, but one of the tests relies on else being None ("create content alert for content & blogPost without title")

Comment (alexduf) [reply src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala]: You're right, then it should be ```keyEvent.title.map(_.webTitle)```

**Review by alexduf — APPROVED** [review]
Looks good!

**Pull request merged**